### ISSUE TITLE
Motors for camjamedu3

### ISSUE THREAD

**Issue by PatrykD01** [CamJam-EduKit/EduKit3#33]
I have tried to get the motors to work but they don't. Is it possible to run them on the raspberry pi zero w v1.1?

**Comment by GeekyTim** [comment]
Can you please give me a little more information? What Raspberry Pi are you using? Have you plugged the motor controller board on top of the Pi with the body overhanging the Pi? Please try a fresh set of batteries.

Yes, this runs well on a Pi Zero W.

**Comment by PatrykD01** [comment]
Im using the raspberry pi zero w with the raspberry pi desktop os. Im mot
sure what im doing wrong but nothing seems to work.

**Comment by PatrykD01** [comment]
The motor controller is connected to the pi and is overhanging the pi. I
have got a fresh set of batteries and it still doesn't work

**Comment by PatrykD01** [comment]
The motor controller is also glowing red

**Comment by GeekyTim** [comment]
That is NOT good. Remove it straight away and check all your connections. Are you able to attach a photo?

**Comment by PatrykD01** [comment]
What does it mean when it's red?

**Comment by PatrykD01** [comment]
Is there a recomended os I can use?

**Comment by GeekyTim** [comment]
When you said it glowed red, I thought it was overheating! I think you meant the LEDs!

The best OS to use is Raspberry Pi OS.

**Comment by PatrykD01** [comment]
Which raspberry pi os (such as desktop)if its ok to ask and thanks for the
replies. Do you know what could cause the motors to not work?

**Comment by GeekyTim** [comment]
You can either use Desktop or Lite.

It is unlikely that the motors themselves are not working as they are simple. To test them, connect the red lead to the + terminal of the battery pack, and the other to the -ve terminal. The motors should spin.

Is it possible that you have connected the battery pack the wrong way on the terminals? A photo of how you have everything connected would really help.

**Comment by PatrykD01** [comment]
. The tips of the battery wires fell of so I'm going to remove the some of
the insulation. I plugged in the right motor(motor a) the other way round
as when I tested the motor straight from the battery pack they rotated
backwards.i attached the code aswell. I'm still unsure of what's wrong and
when I wanted to test to see if the motors will turn when I ran the code I
did have the battery pack plugged in the black wire to the GIND and the red
wire to the VIN port.
